docs/decisions/0031-store-chats-alongside-database.md

```diff
@@ -24,7 +24,7 @@ simple to implement and gives the user the ability to share or not to share the
 
 * Simple implementation
 * The user can send the chats file alongside the `.bib` file if they want to share the chats. If users do not want 
```

Comment:
For markdown there's markdownlint for VsCode https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint
I think there's also an extension for intellih